tools/convert_RbacConfig_to_ClusterRbacConfig.sh

```diff
@@ -2,10 +2,10 @@
 set -e
 set -u
 
-# This script is provided to help converting RbacConfig to ClusterRbacConfig automatically. The RbacConfig
-# will be deleted after the corresponding ClusterRbacConfig is successfully applied.
-# The RbacConfig is deprecated by ClusterRbacConfig due to an implementation bug that could cause the
-# RbacConfig to be namespace scoped in some cases. The ClusterRbacConfig has exactly same specification
+# This script is provided to help converting RbacConfig to ServiceMeshRbacConfig automatically. The RbacConfig
+# will be deleted after the corresponding ServiceMeshRbacConfig is successfully applied.
+# The RbacConfig is deprecated by ServiceMeshRbacConfig due to an implementation bug that could cause the
+# RbacConfig to be namespace scoped in some cases. The ServiceMeshRbacConfig has exactly same specification
 # as the RbacConfig, but with correctly implemented cluster scope.
```

Comment:
I'm not entirely sure about this script. I don't think it makes sense to change it like this, because it doesn't produce the expected outcome: the ServiceMeshRbacConfig will have to live in the CP namespace; this script will just create it wherever the RbacConfig used to be. Also, this last sentence in the comment is now wrong (ServiceMeshRbacConfig is not cluster-scoped).

Reply:
I can either back out these changes or fix the comment to say mesh scoped.  The transition from RbacConfig to ClusterRbacConfig was some time ago so I do not believe we will need to use this script.

Reply:
Do we want to do this here, or in the operator (like we do for most of the changes to the helm charts)?

Reply:
I'll rollback the script changes in the next PR.  As far as the helm chart updates are concerned I think it's better to keep the changes with the code so we have them in a single commit.  We've done this for other changes in the past where code changes were needed but we try to minimise this.